src/components/Checkbox/Checkbox.stories.tsx

```diff
@@ -7,18 +8,96 @@ export default {
   component: Checkbox,
 } as Meta<CheckboxProps>;
 
-export const Default = () => (
-  <Checkbox label="example label" value="1" defaultChecked />
-);
+export const Default = () => {
+  return <Checkbox label="example label" name="example" value="1" />;
+};
+
+export const DefaultChecked = () => {
+  return (
+    <Checkbox label="example label" name="example" value="1" defaultChecked />
+  );
+};
+
+const ControlledTemplate: Story<CheckboxProps> = ({
+  checked: checkedProp = false,
+  ...rest
+}: CheckboxProps) => {
+  const ref = useRef(null);
+  const [checked, setchecked] = useState<"mixed" | boolean>(checkedProp);
```

Comment:
Nit: Should be `setChecked`

```suggestion
  const [checked, setChecked] = useState<"mixed" | boolean>(checkedProp);
```

Reply:
Will change. Would be nice if useStateSnippet takes care of the case.